CHANGELOG.md

```diff
@@ -2,18 +2,19 @@
 
 SECURITY:
 
- * build: Updated to Go 1.14.6. Go 1.14.4 contained 2 CVEs which are low severity for Nomad. [[GH-8467](https://github.com/hashicorp/nomad/issues/8467)]
+ * build: Updated to Go 1.14.6. Go 1.14.5 contained 2 CVEs which are low severity for Nomad [[GH-8467](https://github.com/hashicorp/nomad/issues/8467)]
 
 BUG FIXES:
 
- * jobspec: Fixed validation of multi-region datacenters to allow empty region `datacenters` to default to job-level `datacenters`. [[GH-8426](https://github.com/hashicorp/nomad/issues/8426)]
- * networking: Fixed a bug where a default network interface was not set for group networks if both `network_interface` and `host_network` were omitted. [[GH-8407](https://github.com/hashicorp/nomad/pull/8407)]
```

Comment:
I reworked this one to hopefully explain the behavior seen by users in L10.

Reply:
Much better. Thanks!